gateway/src/constructors/express/express.js

```diff
@@ -4,13 +4,13 @@ const cors = require('cors')
 const morgan = require('morgan')
 const bodyParser = require('body-parser')
 
-module.exports = async ({ io }) => {
+module.exports = async ({ io,...context }) => {
   const app = express()
   app.disable('x-powered-by')
   app.use(morgan('combined'))
   app.use(bodyParser.json({ limit: '50mb', extended: true }))
   app.use(bodyParser.urlencoded({ limit: '50mb', extended: true }))
   app.use(cors())
-  app.use('/', require('./routes')({}))
+  app.use('/', require('./routes')({context}))
```

Comment:
I think we need to pass this without the braces: `app.use('/', require('./routes')(context))`

Reply:
the code is not complete, i  just pushed it bcz , my lap just had some problem, so just pushed the current status to make sure  that i dont loose the work done...will do the changes mentioned in the review